src/utils/hookUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const useClipboard = () => {
+  const copyToClipboard = (data: string) => navigator.clipboard.writeText(data);
```

Comment:
This might not work since `writeText` return a `Promise`. Could you add async/await? https://www.w3.org/TR/clipboard-apis/#dom-clipboard-writetext

Reply:
doesn't matter b/c the arrow fn just returns the promise returned by `writeText` which we can then `await`

Reply:
And depending on the build target `return await` can actually make the code slower (though in this case would be entirely inconsequential). https://arthur.place/the-cost-of-return-await

Reply:
Thanks, @adairrr. @Kiruse already incorporated your change in the new design. Please pull the latest and check it out.